### DISCUSSION TITLE
Prob with api when deploying on other site

### DISCUSSION THREAD

**Discussion by Badaryman** [amvstrm/amvstrm#36] [Discussions]
![Screenshot_2024-01-05-02-15-35-307_com android chrome](https://github.com/amvstrm/amvstrm/assets/154390732/1ed77747-1b2c-4e38-84df-678206f671a5)
![Screenshot_2024-01-05-02-15-41-671_com android chrome](https://github.com/amvstrm/amvstrm/assets/154390732/d6e0bc21-e735-42dc-a1b5-ee3ce939de8d)

https://prnt.sc/ldEpNIA0qJUf

So basically the site im using uses both v1 and v2 of the api to work but when i tried to use my hosted api for v1 and v2 there is an undefined in the url

this is the site

https://test-xi-indol-37.vercel.app/

this is the api
https://api-self-eight.vercel.app/

**Comment by NYT92** [comment]
I think you should edit the package ``@consumet/extensions`` version.

```json
"@consumet/extensions": "github:consumet/consumet.ts"
```

Idk why but I also have similar problems with it but I managed to make it work by changing the npm version to the git version.

**Reply by Badaryman** [reply]
where is that?
btw sorry if i asked every single thing it just that i new to this thing and i understand pretty quick so hope you dont mind

**Reply by NYT92** [reply]
It fine, I already know you struggle abit but it should be in the package.json.

Just edit them out and reinstall the package locally and test it out

**Reply by Badaryman** [reply]
![image](https://github.com/amvstrm/amvstrm/assets/154390732/cf9d7f29-7656-4526-aab2-91f6682659c8)

**Reply by Badaryman** [reply]
just change that right

**Reply by NYT92** [reply]
Find and edit the package ``@consumet/extensions`` version from "^1.**" To "github:consumet/consumet.ts"

```json
"@consumet/extensions": "github:consumet/consumet.ts"
```

**Reply by Badaryman** [reply]
its now working now bro tysm

**Comment by JammyVibez** [comment]
**can anyone reply me please I need more information about the API and I sent àn email but haven't gotten any response**

**Reply by NYT92** [reply]
What can I help you?

**Reply by JammyVibez** [reply]
Does amvstrm has any API? 
And I have few proposal and chat I would like to have with you

**Reply by NYT92** [reply]
We do have API here and it is open-source too.

[API Repo](Https://github.com/amvstrm/API)

**Reply by JammyVibez** [reply]
It's a lucrative proposal....

**Reply by JammyVibez** [reply]
> We do have API here and it is open-source too.
> 
> [API Repo](Https://github.com/amvstrm/API)

Okay

**Comment by JammyVibez** [comment]
Hi
